how-it-works/2_chainkeytech/04_chain_key_tokens.subpage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chain-key Tokens
+abstract:
+shareImage: /img/how-it-works/direct-integration-with-bitcoin.600.jpg
+slug: chain-key-tokens
+---
+
+# Chain-Key Tokens
+
+One important way how different blockchains can cooperate is by bringing tokens from one blockchain to another one, e.g., bringing Bitcoin to the Ethereum chain. In traditional blockchain architectures, this is mostly accomplished through wrapping. More decentralized (and secure) alternatives to wrapping exist and can replace it: Meet *chain-key cryptography* and *chain-key tokens* as explained on this page!
+
+A *wrapped token* represents an underlying asset, which is typically native on a different blockchain than the wrapped token. For example, a wrapped bitcoin token represents real bitcoin, but is a different token than bitcoin and available on a different chain, e.g., the Ethereum or Solana blockchain. Traditional wrapping always involves intermediaries that need to be trusted.
+
+The best known alternative to wrapping tokens is to use advanced threshold cryptography to obtain *chain-key tokens*. Chain-Key Bitcoin, the first major chain-key token on ICP, is a token on ICP with a 1:1 backing with real bitcoin held 100% on chain on ICP by a smart contract. Thus, it is a Bitcoin ‘twin’ on ICP that features low transaction fees and latency and high throughput, similar in properties to a Bitcoin Layer 2. Chain-key token ledgers are compatible with ICP's ICRC-1 token standard and thus can be easily integrated by services, e.g., wallets and DEXs. This makes chain-key tokens on ICP readily available for a wide range of services.
```

Comment:
```suggestion
A more secure alternative to wrapping tokens is to use advanced threshold cryptography to obtain *chain-key tokens*. Chain-Key Bitcoin, the first major chain-key token on ICP, is a token on ICP with a 1:1 backing with real bitcoin held 100% on chain on ICP by a smart contract. Thus, it is a Bitcoin ‘twin’ on ICP that features low transaction fees and latency and high throughput, similar in properties to a Bitcoin Layer 2. Chain-key token ledgers are compatible with ICP's ICRC-1 token standard and thus can be easily integrated by services, e.g., wallets and DEXs. This makes chain-key tokens on ICP readily available for a wide range of services.
```

Reply:
If we feel like bragging, we could also say "superior".

Reply:
"on ICP"

I think we should not get into the habit of writing that. It makes little sense. Nobody says "I sent a transaction on BTC".

Reply:
"on ICP" three times in the same sentence!   :-D

Reply:
"Chain-key token ledgers are compatible with ICP's ICRC-1 token standard"

Technically, there is on connection. You can certainly have chain-key tokens on ledgers that do not follow this standard.